app/views/people/_form.html.erb

```diff
@@ -10,32 +11,19 @@
     <%= f.input :status, :as => :select, :collection => Person::STATUS %>
     <%= f.input :end_date, :as => :string, :input_html => { :class => 'datepicker', :size => 11 }  %>
     <%= f.input :department, :as => :select, :collection => Person::DEPARTMENT %>
-    <div class="form-horizontal">
-        <h2> Phone information </h2>
-        <%= f.simple_fields_for :phones do |c| %>
-          <fieldset>
-            <%= c.input :category, :as => :select, :collection => Channel::CATEGORIES %>
-            <%= c.input :content, :placeholder => '9785551212' %>
-            <%= c.input :name, :label => 'Description' %>
-            <%= c.input :status, :as => :select, :collection => Channel::STATUSES %>
-            <%= c.input :usage, :as => :select, :collection => Channel::USAGES %>
-            <%= c.input :carrier, :as => :select, :collection => Phone::CARRIERS %>
-            <%= c.input :sms_available %>
-            <%= c.input :priority, :as => :select, :collection => Phone::PRIORITIES %>         
-          </fieldset>
+
+    <div>
+    <h2> Contact information </h2>
+        <%= f.simple_fields_for :channels do |channel| %>
```

Comment:
I think, if I've interpreted the documentation correctly, that it should be "<%=". From plataformatec: "Wrapper to use Simple Form inside a default rails form. It works in the same way that the fields_for Rails helper." Then the Rails guide section on form helpers has "<%= fields_for @person.contact_detail do |contact_details_form| %>" as an example. I don't quite understand the comment about the <br/>. Could you explain a little bit more what you mean? I know I inserted it so that each "Contact Information" section would have its own area ... do you mean the indentation within the code itself, or the way it presents the data on the screen?